scripts/combine_all_cr.sh

```diff
@@ -0,0 +1,16 @@
+directories=$(ls -l ./charts/ | grep ^d | awk '{print $9}')
+for directory in $directories
+do
+	echo -e "---\n" > ./charts/$directory/experiments.yaml
+	subDirectories=$(ls -l ./charts/$directory | grep ^d | awk '{print $9}')
```

Comment:
this is fine, but would you like to explore better ways to getting the subdir list?